README.md

```diff
@@ -270,6 +272,20 @@ export SPHERE_CLIENT_SECRET="your-client-secret"
 #### ElasticIo
 _(Coming soon)_
 
+#### UserAgent
+A synchronous module that builds _user\_agent_ according to the standard specified [here](https://sphere.atlassian.net/wiki/x/7ACjB)
+
+sphere-node-sdk module must be installed in the node_modules because it's required to build the user_agent
+
+```js
+const user_agent = userAgent('sphere-node-utils', '1.0.0')
```

Comment:
`user_agent` > `userAgent`.
I know that the SDK doesn't follow camel case but we should as far as we can.